tests/test_scripts.py

```diff
@@ -0,0 +1,17 @@
+import tensorflow as tf
+
+from imitation.scripts.policy_eval import policy_eval_ex
+from imitation.scripts.train import train_ex
+
+
+def test_policy_eval():
+  """Smoke test for imitation.scripts.policy_eval"""
+  run = policy_eval_ex.run(config_updates={'render': False},
+                           named_configs=['fast'])
+  assert run.status == 'COMPLETED'
+  assert isinstance(run.result, dict)
+
+
+def test_train():
+  train_ex.run(named_configs=['cartpole', 'gail', 'debug'])
+  tf.reset_default_graph()
```

Comment:
It might be annoying to have test sacred logs clutter the log dir whenever I run `pytest tests/`. Do you know of a way to easily separate them from actual logs? Probably still want to save them for debugging purposes. Maybe we could save them into `output/sacred_test` instead of `output/sacred`.

Anyhow this shouldn't block this PR.

Reply:
The `FileStorageObserver` isn't attached (since it gets added in the `if name == '__main__'` guarded block), so no Sacred output will be produced. However, it will still produce output such as the checkpoints and other non-Sacred logging info. I've changed the `log_root` so this goes into `output/tests` now.